### DISCUSSION TITLE
B2B Company Role jstree

### DISCUSSION THREAD

**Discussion by web01bharat** [breezefront/community#150] [1. Help]
Hi support,

We are using Magento native feature B2B. In the admin, as we go to the Company Role edit page its designs do not work as Luma seems its tree-js and CSS not working.

Luma design

![Screenshot Capture - 2023-06-08 - 17-28-09](https://github.com/breezefront/community/assets/135975409/248dc3a9-1685-4797-98e4-2e3e2bf5c98f)

Breeze design
![treejs](https://github.com/breezefront/community/assets/135975409/5fb2c4cc-526f-4cba-9857-ad157ca6c9a9)

Please let me know how we can achieve it as we have luma.

Thanks
Bharat

**Comment by vovayatsyuk** [comment]
Breeze doesn't support this feature (I doubt if we support any of the B2B features).

**Reply by sandipansaha** [reply]
Hi @vovayatsyuk Is there any future plan for the B2B features support? If required I can provide you access to the B2B modules for the development.
Thanks
